Ginger/Ginger/CustomTraceListener.cs

```diff
@@ -0,0 +1,572 @@
+#region License
```

Comment:
Use Ginger logs for logging

Reply:
this is for alm repository listener is up or not